rules/linux/process_creation/proc_creation_lnx_password_group_discovery.yml

```
@@ -0,0 +1,27 @@
+title: Detection of Password Groups Discovery
+id: 2b7d6524-c142-435a-ae96-be23543b9883
+status: test
```

```suggestion
status: experimental
```
https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-rule-convention.md